worlds/sm/Options.py

```diff
@@ -239,6 +244,94 @@ class VariaCustomPreset(OptionList):
+    You then have to find your way to the ship in the remaining time. Allotted time depends on area layout, but not on skill settings and is pretty generous.
+    
+    During the escape sequence:
+    - All doors are opened
+    - Maridia tube is opened
+    - The Hyper Beam can destroy Bomb , Power Bomb  and Super Missile  blocks and open blue/green gates from both sides
+    - All mini bosses are defeated
+    - All minor enemies are removed to allow you to move faster and remove lag
+
+    During regular game only Crateria Map station door can be opened and activating the station will act as if all map stations were activated at once.
+
+    Animals Challenges:
+    You can use the extra available time to:
+    - find the animals that are hidden behind a (now blue) map station door
+    - go to the vanilla animals door to cycle through the 4 available escapes, and complete as many escapes as you can
+
+    Pick your challenge, or try to do both, but watch your timer!
+    """
+    display_name = "Randomize the escape sequence"
+
+class RemoveEscapeEnemies(Toggle):
+    """Remove enemies during escape sequence, disable it to blast through enemies with your Hyper Beam and cause lag."""
+    display_name = "Remove enemies during escape"   
+
+class Tourian(Choice):
+    """
+    Choose endgame Tourian behaviour:
+    Vanilla: regular vanilla Tourian
+    Fast: speed up Tourian to skip Metroids, Zebetites, and all cutscenes (including Mother Brain 3 fight). Golden Four statues are replaced by an invincible Gadora until all objectives are completed.
+    Disabled: skip Tourian entirely, ie. escape sequence is triggered as soon as all objectives are completed.
+    """
+    display_name = "Endgame behavior with Tourian"
+    option_Vanilla = 0
+    option_Fast = 1
+    option_Disabled = 2
+    default = 0  
+
+class Objective(OptionList):
+    """
+    Choose which objectives are required to sink the Golden Four statue and to open access to Tourian.
+    You can choose from 0 to 5 objectives.
+    Note: If you leave the list empty no objective is required to access Tourian, ie. it's open.
+    Note: See the Tourian parameter to enable fast Tourian or trigger the escape when all objectives are completed.
+    Note: Current percentage of collected items is displayed in the inventory pause menu.
+    Note: Collect 100% items is excluded by default when randomizing the objectives list as it requires you to complete all the objectives.
+    Note: In AP, Items% and areas objectives are counted toward location checks, not items collected or received, except for "collect all upgrades"
+
+    Format as a comma-separated list of objective names: ["kill three G4", "collect 75% items"].
+    A full list of supported objectives can be found at:
+    https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/sm/utils/objectives.py
+    """
+    display_name = "Objectives"
+    default = "kill all G4"
```

Comment:
I think this needs to be `default = ["kill all G4"]`? What this puts in the yaml template leads to generation failure